lib/widgets/screens/sign_in/sign_in_screen.dart

```diff
@@ -64,161 +65,342 @@ class _SignInScreenState extends State<SignInScreen> {
+                                    } else {
+                                      router.push('/');
                                     }
-                                  } else {
-                                    router.push('/');
                                   }
-                                }
-                              },
-                              child: Padding(
-                                padding: const EdgeInsets.all(8.0),
-                                child: Text(
-                                  'Sign In',
-                                  style: GoogleFonts.poppins(
-                                    fontSize: 14,
-                                    fontWeight: FontWeight.bold,
+                                },
+                                child: Padding(
+                                  padding: const EdgeInsets.symmetric(
+                                    vertical: Insets.widgetSmallInset,
+                                    horizontal: Insets.widgetLargeInset * 2.5,
+                                  ),
+                                  child: Text(
+                                    l10n.logIn,
+                                    style: TextStyles.largeTextButton(context),
                                   ),
                                 ),
                               ),
                             ),
+                          ],
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(Insets.widgetSmallInset),
+                      child: TextButton(
+                        key: const Key('registerButton'),
+                        onPressed: () {
+                          _openSignUpScreen(context);
+                        },
+                        child: Text(
+                          l10n.createNewAccount,
+                          style: TextStyles.sectionSubHeaderBold(context),
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(Insets.widgetSmallInset),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          Container(
+                            width: Dimensions.loginBoxWidth / 2,
+                            height: Dimensions.lineHeight,
+                            color: Colors.black,
+                          ),
+                          Text(
+                            l10n.or,
+                            style: TextStyles.sectionSubNote(context),
+                          ),
+                          Container(
+                            width: Dimensions.loginBoxWidth / 2,
+                            height: Dimensions.lineHeight,
+                            color: Colors.black,
```

Comment:
Use the colorScheme colors instead of hardcoding them.

Reply:
Thanks for the comment. I'll make that change.